src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,110 @@
+package mate.jdbc.dao;

```

Comment:
let's put your implementations into 'impl' package. Do the same with ManufacturerDaoImpl